docs/docs/upgrade/network-api-migration.md

```diff
@@ -0,0 +1,707 @@
+---
+title: Migrating Tools to the 1.5 Networking API
+---
+
+# Migrating Tools to the 1.5 Networking API
+
+## Overview
+The networking section of the Marathon API has changed significantly in version 1.5. Marathon can still accept requests using the 1.4 version of the API, but it will always reply with the 1.5 version of the app definition.
+
+This WILL likely break tools that consume networking-related fields of the application definition.
+
+This document contains the high-level structural differences between the 1.4 and 1.5 API versions. You can read more details regarding the networking section and the correct values for every field in the [networking documentation]({{ site.baseurl }}/docs/networking.html).
+
+### Important notes
+
+Marathon now supports applications attached to more than one network. However, additional rules apply when you specify more than one network:
+
+- If a *portMapping* has defined a `hostPort`, the `networkNames` array must be defined with the name(s) of the `networks` it's referring to.
```

Comment:
... and `networkNames` is only required to be defined for this case when there are more than 1 container networks declared